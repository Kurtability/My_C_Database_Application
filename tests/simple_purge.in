purge a
get a
set a 1 2 3
purge a
list keys
list entries
set b 1
list entries
purge b
get b
list entries
bye
