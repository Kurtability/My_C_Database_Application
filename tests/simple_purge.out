> purge a 
ok

> get a 
no such key

> set a 1 2 3 
ok

> purge a
ok

> list keys
no keys

> list entries 
no entries

> set b 1
ok

> list entries
b [1]

> purge b
ok

> get b
no such key

> list entries 
no entries

> bye
bye
